modules/Search/components/SearchBar.module.scss

```diff
@@ -0,0 +1,77 @@
+.wrapper {
```

Comment:
It seems strange that search input is so small on desktop (even smaller than on mobile)

<img width="1357" alt="image" src="https://user-images.githubusercontent.com/1641218/180999545-360b15c1-4c5e-4f07-9375-372e809cde70.png">